plugins/versionpress/src/Utils/AutocompleteUtils.php

```diff
@@ -102,4 +102,18 @@ private static function getDateConfig()
             'type' => 'date'
         ];
     }
+
+    /**
+     * @param $string
+     * @return string
+     */
+    private function capitalize($string)
+    {
+        $capitalizedString = Strings::firstUpper($string);
+        return str_replace(
+            ['Wordpress', 'wordpress', 'Versionpress', 'versionpress'],
```

Comment:
The impl seems too specific. Wouldn't it be better to match "wordpress" and "versionpress" case-insensitively and replace with proper forms?